Automattic/vip-go-mu-plugins
# Updating Akismet to 4.2.1

## Description

This PR fetches the latest version of the Akismet plugin, 4.2.1.

## Changelog Description

### Plugin Updated: Aksimet 4.2.1

We upgraded Akismet 4.1.13 to Akismet 4.2.1.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Make sure Jetpack and Akismet are enabled.
3. Check that the Akismet version is 4.2.1
4. Perform the viagra test.
